projects/pnetwork.js

```diff
@@ -0,0 +1,11 @@
+const axios = require('axios')
+
+async function fetch() {
+    let a = await axios.get('https://pnetwork.watch/api/datasources/proxy/1/query?db=pnetwork-volumes-1&q=SELECT%20%22tvl%22%20FROM%20%22tvl%22%20WHERE%20time%20%3E%3D%20now()%20-%201d&epoch=ms')
+    return a.data.results[0].series[0].values.slice(-1)[0][1]
+}
+
+module.exports = {
+    fetch,
```

Comment:
The TVL for pTokens should be broken down into each chain since the TVL is split across 7 chains.

- bsc: $38.6M
- eos: $6.83M
- eth: $135M
- polygon: $87.8K
- telos: $517K
- ultra: $1.84M
- xdai: $2.33K

Total: $183M